### DISCUSSION TITLE
Microsoft Windows 10" Fake Logon Screen

### DISCUSSION THREAD

**Discussion by mgoughnorge** [community/community#38497] [Other Feature Feedback, Questions, & Ideas] [answered]
Love this payload idea.  Very new to Bash Bunny.
Could anyone point me where I can find info on how to setup for these lines in the payload?  Just asking for a pint in the right direction. 

From "payload.txt" change the values of the following constants :

######## INITIALIZATION ########

readonly BB_LABEL="BashBunny"
readonly DROP_URL="http://evil.corp:8080/drop.php?ZXZpbC5jb3Jw.png"
readonly INPUT_PLACEHOLDER="Password"

**Comment by david-engelmann** [comment, marked as answer]
[You will get better responses posting it in this repository](https://github.com/hak5/bashbunny-payloads). [Their twitter could also help you out](https://twitter.com/intent/tweet?url=https%3A//hackinglab.cz/en/blog/bash-bunny-guide/)!

**Comment by mgoughnorge** [comment]
Thank you for your response.  Appreciate it :)

On Mon, Nov 7, 2022 at 10:40 AM David Engelmann ***@***.***>
wrote:

> You will get better responses posting it in this repository
> <https://github.com/hak5/bashbunny-payloads>. Their twitter could also
> help you out
> <https://twitter.com/intent/tweet?url=https%3A//hackinglab.cz/en/blog/bash-bunny-guide/>
> !
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/38497#discussioncomment-4079518>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A37CEB2VU3GAV6CL7CY7LFDWHFEJ7ANCNFSM6AAAAAARZPKLOQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Mike GOUGH

**Reply by david-engelmann** [reply]
No problem, feel free to Mark it as answer to close it out!

**Reply by david-engelmann** [reply]
@mgoughnorge Usually people mark the response with the answer in it as "Marked as answer"

**Comment by mgoughnorge** [comment]
Sorry, Guess i could say I'm not new, but am embarrassingly new (eye roll
allowed)

On Mon, Nov 7, 2022 at 11:06 AM David Engelmann ***@***.***>
wrote:

> @mgoughnorge <https://github.com/mgoughnorge> Usually people mark the
> response with the answer in it as "Marked as answer"
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/38497#discussioncomment-4079721>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A37CEB6RBBIQZR4LT3YN6F3WHFHJTANCNFSM6AAAAAARZPKLOQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Mike GOUGH

**Reply by david-engelmann** [reply]
No need to apologize! Thank you for switching it!
